modules/configuration/src/main/java/org/apache/ignite/internal/configuration/ConfigurationChanger.java

```diff
@@ -100,73 +105,58 @@
         private final SuperRoot roots;
 
         /** Version associated with the currently known storage state. */
-        private final long version;
+        private final long ver;
```

Comment:
What's the reason for shortening the name? If anything, you should have renamed it to "changeId" :)